Gemfile

```diff
@@ -65,6 +70,11 @@ group :development, :test do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+
+  gem 'rspec-rails'
+
+  gem 'factory_girl_rails'
```

Comment:
`rspec-rails` and `factory_girl_rails` should go to `development` and `test` groups. (See the [article](https://semaphoreci.com/community/tutorials/setting-up-the-bdd-stack-on-a-new-rails-4-application).)